client/my-sites/domains/domain-management/edit/style.scss

```diff
@@ -98,6 +98,14 @@
 					font-size: 16px;
 				}
 			}
+
+			a {
+				color: currentColor;
+
+				.gridicons-external {
+					margin-left: 4px;
```

Comment:
Can we move the icon down by 1 pixel so that it lines up with the bottom of the copy?

You can do that with by setting `transfrom` to `translateY(1px)`